server/routerlicious/packages/routerlicious/src/scribe/index.ts

```diff
@@ -51,6 +52,11 @@ export async function scribeCreate(config: Provider): Promise<IPartitionLambdaFa
         },
         true);
 
+    if (createCosmosDBIndexes) {
+        await scribeDeltas.createIndex({ "operation.sequenceNumber": 1 }, false);
+        await scribeDeltas.createTTLIndex({_ts:1}, mongoExpireAfterSeconds);
```

Comment:
Should we call "createTTLIndex" always or only when mongoExpireAfterSeconds > 0, as below?

Reply:
Ah so the reason we call mongoExpireAfterSeconds > 0 below is to decide whether this TTL expiry is turned on or not. In OSS we currently set mongoExpireAfterSeconds to -1 by default so that we never create TTL index for the locally running mongo instance.

For CosmosDB, since we know we will always need to have the TTL expiry for compliance reasons, and we already have the createCosmosDBIndexes config value check (which we can enable locally but set to false in OSS), I thought we do not need to again check for mongoExpireAfterSeconds > 0. What do you think?

Reply:
According to the documentation, there seems to be no problem in setting the TTL to -1 in CosmosDB: https://docs.microsoft.com/en-us/azure/cosmos-db/time-to-live#time-to-live-for-containers-and-items

The document doesn't seem to say it, but I believe that if the value is set to 0, the data would be immediately expired.
So I think this might not be a big deal.